doc/user/zebra.rst

```diff
@@ -351,6 +360,143 @@ default) should the specified gateways not be reachable. Eg:
+network namespace` where Zebra started.
+
+If the -n option is chosen, then the `Linux network namespace`
+will be mapped over the Zebra VRF. That implies that Zebra is
+able to configure several `Linux network namespaces`.  The routing
+table associated to that VRF is the whole routing tables located
+in that namespace. For instance, this mode matches Opentack or 6WIND
+Linux vRouters ; others have reported to use this mode too. The
+default behavior remains Linux VRF lite that os widely used by Cumulus
+Linux.
+
+Because of that difference, there are some subtle differences when running
+some commands in relationship to VRF. Here is an extract of some of those
+commands:
+
+.. index:: vrf VRF
+.. clicmd:: vrf VRF
+
+    This command is available on configuration mode.
+    By default, above command permits accessing the vrf configuration mode.
+    This mode is available for both VRFs. It is to be noted that Zebra does
+    not create `Linux VRF`.
+    Provisioning this command is used to keep the configuration intact.
+
+.. index:: netns NAMESPACE
+.. clicmd:: netns NAMESPACE
+
+    This command is based on VRF configuration mode.
+    This command is available when Zebra is run in `-n` mode.
+    This command reflects which `Linux network namespace` is to be
+    mapped with Zebra VRF. It is to be noted that Zebra creates
+    and detects added/suppressed VRFs from the Linux environment
+    (in fact, those managed with iproute2).
+    Provisioning this command is used to keep the configuration intact.
+
+.. index:: ip route NETWORK NETMASK GATEWAY
+.. clicmd:: ip route NETWORK NETMASK GATEWAY
+
+    This command is based on VRF configuration mode or in configuration mode.
+    If applied on configuration mode, this applies to default VRF. Otherwise,
+    this command applies to the VRF of the vrf configuration mode.
+    By default, above command permits configuring a network in the table
+    identifier of the mentioned VRF.
+    If `-n` option is given, the network will be configured in the default
+    routing table of the `Linux network namespace` <VRF>.
+
+.. index:: ip route NETWORK NETMASK GATEWAY NEXTHOPVRF
+.. clicmd:: ip route NETWORK NETMASK GATEWAY NEXTHOPVRF
+
+    This command is based on VRF configuration mode or in configuration mode.
+    If applied on configuration mode, this applies to default VRF. Otherwise,
+    this command applies to the VRF of the vrf configuration mode.
+    This command is used to configure a vrf route leak across 2 VRFs.
+    This command is only available when Zebra is launched without `-n` option
+
+.. index:: ip route NETWORK NETMASK GATEWAY table TABLENO
+.. clicmd:: ip route NETWORK NETMASK GATEWAY table TABLENO
+
+    This command is based on VRF configuration mode. There, this command is only
+    available with `-n` command. This commands permits configuring a network
+    route in the given <TABLENO> of the `Linux network namespace` <VRF>.
```

Comment:
Remove `<VRF>` and replace `<TABLENO>` with
```
``TABLENO``
```

Reply:
ok